standalone-node/installation_scripts/bootable-usb-prepare.sh

```diff
@@ -227,6 +229,7 @@ partitions_setup() {
     END_MB=$(echo "$START_MB + $OS_IMG_PARTITION_SIZE" | bc)
     create_partition "${START_MB}MB" "${END_MB}MB" "OS image storage" || return 1
     create_partition "$(sudo parted "$USB_DEVICE" -ms print | tail -n 1 | awk -F: '{print $3}' | tr -d 'MB')MB" "${K8S_PARTITION_SIZE}" "K8 storage" || return 1
+    create_partition "$(sudo parted "$USB_DEVICE" -ms print | tail -n 1 | awk -F: '{print $3}' | tr -d 'MB')MB" "${GUEST_VM_SIZE}" "Guest VM storage" || return 1
```

Comment:
If the previous partition creation fails, this may create overlapping or misaligned partitions. Check the result of each create_partition before proceeding to the next.